src/components/Layout.tsx

```diff
@@ -1,14 +1,12 @@
-import React from "react";
-
 import AppBar from "@material-ui/core/AppBar";
-import Toolbar from "@material-ui/core/Toolbar";
-
 import {
-  withStyles,
-  WithStyles,
   createStyles,
   Theme,
+  withStyles,
+  WithStyles,
 } from "@material-ui/core/styles";
+import Toolbar from "@material-ui/core/Toolbar";
+import React from "react";
```

Comment:
Let's keep it consistent: 

```typescript
import React from "react"
import Any from "outsidelib"
import { named } from "outsidelib"

import AnyPersonal from "./my/path"
import { named 2 } from "./my/path
```

- Default imports from outside libraries
- Named imports from outside libraries
- Default imports from local project
- Named imports from local project